bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -794,6 +796,19 @@ public void onFailure(Throwable cause) {
         return op.future();
     }
 
+    /*
+     * Read the last entry in the ledger
+     *
+     * @param cb
+     *            object implementing read callback interface
+     * @param ctx
+     *            control object
+     */
+    public void asyncReadLastEntry(ReadCallback cb, Object ctx) {
```

Comment:
Can you remove this from this pull request and create a seperate one for it?

- we need make this as part of new api as well
- we need a test case to cover it.

Reply:
It's unused it seems. Removing completely for now.